inst/use_cases/seroprevalence.Rmd

```diff
@@ -0,0 +1,64 @@
+---
+title: "Example use case: the importance of clear reporting of seroprevalence estimates"
+output: html_document
+date: "2024-09-23"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(matrixStats)
+library(binom)
+options(scipen=999)
+```
+
+```{r,include=FALSE}
+# essentially we have a scenario in which the neutralisation tests are not reported with the correct denominator and it implies a higher seroprevalence than it actually is, which in turn could suggest a smaller susceptible population 
+
+# set up some numbers
+total_tests <- 5000
+positive_elisa <- 300
+positive_neutralisation <- 50
+
+## could then also talk about if this was just HCWs but that seems patronising?
+```
+
+Estimates of seroprevalence provide critical insights into the level of susceptibility within a population, in turn informing the implementation of control measures, including vaccinations (e.g. through the critical fraction requiring vaccination to control spread). \
+A novel coronavirus has been identified and is spreading throughout the population. A rapid seroprevalence study is undertaken to understand levels of immunity of the population. Initially, `r total_tests` tests are carried out via enzyme-linked immunoabsorbsent assay (ELISA), of which `r positive_elisa` are positive. This corresponds to a seroprevalence estimate of `r round(binom.confint(x = positive_elisa,n = total_tests,method="exact")$mean,3) * 100`% (95% exact binomial confidence intervals `r round(binom.confint(x = positive_elisa,n = total_tests,method="exact")$lower,3) * 100`% - `r round(binom.confint(x = positive_elisa,n = total_tests,method="exact")$upper,3) * 100`%). \
+However, it is known that ELISA assays can be prone to cross-reactivity with other coronaviruses, which are also in circulation in this population. Therefore, it is decided to undertake neutralisation tests, which are typically more sensitive, to provide further confidence in the level of population-immunity. Due to a limited budget, only `r 2*positive_elisa` tests can be re-tested with a neutralisation assay, and so all of the `r positive_elisa` positive tests and `r positive_elisa` of the `r total_tests - positive_elisa` negatives are selected randomly for further testing.\
+Of the `r 2*positive_elisa` samples sent for further testing, only `r positive_neutralisation` return as positive. For the neutralisation tests, in respect of the ELISA tests, this corresponds to a seroprevalence estimate of `r round(binom.confint(x = positive_neutralisation,n = 2*positive_elisa,method="exact")$mean,3) * 100`% (95% exact binomial confidence intervals `r round(binom.confint(x = positive_neutralisation,n = 2*positive_elisa,method="exact")$lower,3) * 100`% - `r round(binom.confint(x = positive_neutralisation,n = 2*positive_elisa,method="exact")$upper,3) * 100`%), which is notably lower than the seroprevalence obtained under ELISA only. \
```

Comment:
I see a "seroprevalence estimate of 6%" for the ELISA and a "a seroprevalence estimate of 8.3%" for the neutralisation test. So I think this sentence should say higher not lower.

Reply:
good catch - thanks! I also changed the figures so they're more different to underline the point